tests/api_tests/test_health.py

```diff
@@ -23,15 +23,6 @@ def tearDown(self) -> None:
         except MarqoApiError as s:
             pass
 
-    def test_health_deprecation_warning(self):
-        with mock.patch("marqo.client.mq_logger.warning") as mock_warning:
-            res = self.client.health()
-
-            # Check the warning was logged
-            mock_warning.assert_called_once()
-            warning_message = mock_warning.call_args[0][0]
-            self.assertIn("The `client.health()` API has been deprecated and will be removed in", warning_message)
-
```

Comment:
This functionality is py-marqo specific and out of scope for the API test suite. The equivalent test can be found in py-marqo here: https://github.com/marqo-ai/py-marqo/blob/mainline/tests/v0_tests/test_client.py#L65

Reply:
Thanks for the information!